packages/attribute-registry-process/src/routers/AttributeRouter.ts

```diff
@@ -174,20 +158,9 @@ const attributeRouter = (
             unsafeBrandId(req.params.attributeId)
           );
 
-          if (attribute) {
-            return res.status(200).json(toApiAttribute(attribute.data)).end();
-          } else {
-            const errorRes = makeApiProblem(
-              attributeNotFound(req.params.attributeId),
-              getAttributeByIdErrorMapper
-            );
-            return res.status(errorRes.status).json(errorRes).end();
-          }
```

Comment:
Refactor. There is no need to throw `attributeNotFound` inside the router. It can be thrown inside `attributeRegistryService`.